vnext/Microsoft.ReactNative/RedBoxErrorFrameInfo.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "RedBoxErrorFrameInfo.h"
+#include "unicode.h"
+
+namespace Mso::React {
```

Comment:
can we please put this and RedBox in the Microsoft::ReactNative namespace instead of Mso::React?

Reply:
I don't think I want to do this with this change. This code is shared with the UWP DLL and would likely require modifications beyond what I want to accomplish with this change (which is to achieve ABI parity between the UWP and the Win32 DLL). FWIW, this class is only used internally; the public ABI counterpart is Microsoft::ReactNative::IRedBoxErrorFrameInfo, i.e. it is in the namespace you're asking for.